### DISCUSSION TITLE
[Feature] Added Image collecting

### DISCUSSION THREAD

**Discussion by Gamrin** [immich-app/immich#6488] [Feature Request]
### The feature

![image](https://github.com/immich-app/immich/assets/4996362/81585ccc-1bef-4317-88d1-e5a983a39090)

When uploading images to an album, if I drag any amount of files into the browser, I get a separate "added 1 asset" popup for every single one of them. This is not only very visually annoying, but it also means 

Suggestion: Collect these by the grouping with which they were added to the queue.

Paste in 100 files, get "added 100 assets" as a popup when they are done. Or "added 95 assets" if there's already some duplicates.

This also blocks the upload window, and it blocks the "new album" button.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by samholton** [comment]
Cannot reproduce in v1.98.1
